courses/intro-to-ada/chapters/privacy.rst

```diff
@@ -164,3 +164,174 @@ Ada also allows you to implement special semantics for assignment via
 assignment is simply inappropriate; one example is the :ada:`File_Type` from the
 :ada:`Ada.Text_IO` package, which is declared as a limited type and thus
 attempts to assign one file to another would be detected as illegal.
+
+Child packages & privacy
+------------------------
+
+We've seen previously (in the :ref:`child packages section <ChildPackages>`)
+that packages can have child packages. Privacy plays an important role in child
+packages. This section discusses some of the privacy rules that apply to child
+packages.
+
+Child packages have access to the private part of their parent's specification:
```

Comment:
Not fully correct. Child package bodies have access; in the spec, only the private part of the child spec has access to the private part of the parent (unless the child is a private subunit)